docs/gettingStarted/customScheme/developCustomUniScheme.md

```diff
@@ -1,4 +1,5 @@
 ## Design Principle
+*Note: The Universal Schemes approach is deprecated and will be phased out in the future. It is now recommended to use a [Non-Universal Scheme](../developCustomNonUniScheme/)*
```

Comment:
it is deprecated for arc version > rc.33

Reply:
It's generally deprecated, that is, you shouldn't create new schemes like that. In v33 we just removed UGenericScheme and UController.